positron/modules/positron_util.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { classes: Cc, interfaces: Ci, results: Cr, utils: Cu } = Components;
+
+exports.makeStub = function(signature, returnValue) {
```

Comment:
> Sorry to be a pain, but could we make "returnValue" an object literal instead? The reason being that in various places it looks like a boolean trap. It will also make the API a little more legible + extensible if need be in the future. So:

Ok, I've made the second parameter an object literal with a returnValue property in 00e6c98.